client/src/components/History/CurrentHistory/HistoryNavigation.vue

```diff
@@ -6,17 +6,33 @@
         <nav class="d-flex justify-content-between mx-3 my-2">
             <h4 class="m-1">History</h4>
             <b-button-group>
+                <b-button
+                    data-description="create new history"
+                    size="sm"
+                    variant="link"
+                    v-b-tooltip.bottom.hover
+                    title="Create new history"
+                    @click="$emit('createNewHistory')">
+                    <Icon fixed-width icon="plus" />
+                </b-button>
+
+                <b-button
+                    v-b-modal.history-selector-modal
+                    data-description="show saved histories"
+                    size="sm"
+                    variant="link"
+                    v-b-tooltip.bottom.hover
+                    title="Show saved histories">
+                    <Icon fixed-width icon="list" />
+                </b-button>
```

Comment:
Maybe something like this to make it more obvious it is for quick changing the current History:
```suggestion
                <b-button
                    v-b-modal.history-selector-modal
                    data-description="switch to another history"
                    size="sm"
                    variant="link"
                    v-b-tooltip.bottom.hover
                    title="Switch to history">
                    <Icon fixed-width icon="exchange-alt" />
                </b-button>
```